functions/startup/routes.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const categories = require('../api/category');
+const products = require('../api/product');
+const operations = require('../api/operation');
+const thresholds = require('../api/threshold');
+const units = require('../api/unit');
+const httperror = require('../middleware/httperror');
+module.exports = function(app) {
+    app.use(express.json());
+    app.use(express.urlencoded({extended: true}));
+    app.use('/api/categories', categories);
+    app.use('/api/products', products);
+    app.use('/api/operations', operations);
+    app.use('/api/thresholds', thresholds);
+    app.use('/api/units', units);
```

Comment:
Can we just expose as /categories without the **api**

Reply:
umm, you can remove the **api** and have it as just **/categories**. in index.js file,
exports.**app** = functions.https.onRequest(app) 
replace the above line with
exports.**api** = functions.https.onRequest(app)

Reply:
Done.